controllers/system/system_controller.go

```diff
@@ -1102,6 +1102,19 @@ func (r *SystemReconciler) ReconcileRequired(instance *starlingxv1.System, spec
 		return err, false
 	}
 
+	var res starlingxv1.CertificateList
+	certificateMap := make(map[string]bool)
+
+	for _, eachCertificate := range *spec.Certificates {
```

Comment:
eachCertificate  can be renamed as "certificate" or "cert"

Reply:
updated